packages/mui-base/src/Slider/Control/SliderControl.tsx

```diff
@@ -71,6 +71,12 @@ const SliderControl = React.forwardRef(function SliderControl(
   return renderElement();
 });
 
+export namespace SliderControl {
+  export interface Props extends BaseUIComponentProps<'span', SliderRoot.OwnerState> {}
```

Comment:
```suggestion
  export interface OwnerState extends SliderRoot.OwnerState {}
  
  export interface Props extends BaseUIComponentProps<'span', OwnerState> {}
```

+ other cases where the OwnerState isn't defined explicitly